subprojects/libero-core/src/main/groovy/com/connexta/libero/Libero.groovy

```diff
@@ -204,7 +256,14 @@ class Libero {
 
         git.checkout(branch: config.originalBranch)
 
-        println "Release Process Complete!!"
+        log.info("Release Process Complete!!")
+        if (options.dryRun) {
+            log.info("Remember this was a dry run!! You should clean up once finished with it. To clean up: recommend " +
+                    "running 'git reset --hard ${config.sourceRemote}/${config.ref}' and 'git tag -d ${config.releaseName}")
```

Comment:
I would expect a dry run to clean up after itself on success or failure.  What does maven release do after a dry run?